### ISSUE TITLE
Department of Renaming: Move other insts in.

### ISSUE THREAD

**Issue by Artoria2e5** [AOSC-Dev/LiveKit-Installer#1]
Performing git mv.

**Comment by Artoria2e5** [comment]
Ah I think I should merge other installers in instead.

**Comment by Artoria2e5** [comment]
@MingcongBai @Trickness @lmy441900 @bobcao3 @Icenowy

**Comment by jyhi** [comment]
No. Separated repositories means these installers are different from each other.

**Comment by Artoria2e5** [comment]
@lmy441900 No. We don't need the difference.

**Comment by Artoria2e5** [comment]
And neither will I move you AST in. @lmy441900

**Comment by jyhi** [comment]
@Arthur2e5 

> We don't need the difference.

In spite of this, the history of the project should be kept, and in separated repositories.

I know you guys just want to create orphan branches to store them so that there will be fewer inactive repositories. I do not suggest since these installers are based on different technologies, but if you really want to do this, I have no objection.

> And neither will I move you AST in.

I know you won't do this.

**Comment by bobcao3** [comment]
So what about the installer which based on Calamares? (Maybe I spell it wrong, who cares)

**Comment by Artoria2e5** [comment]
@bobcao3 As long as it does the same thing.
@lmy441900 Are you blind? Please look at the history of the files. They are all there!

**Comment by MingcongBai** [comment]
> @lmy441900 Are you blind? Please look at the history of the files. They are all there!

It is rude to just edit your comment... but... well, he's got the message I guess.

**Comment by bobcao3** [comment]
Well, I think using submodules is better.

**Comment by jyhi** [comment]
Anyway I don't support it.

**Comment by Artoria2e5** [comment]
@bobcao based on computions on the repo size and expected growth, they are all small enough for resting in the same repo. This is related to the original Qt/ layout, too, which suggested there should be something else.
